### DISCUSSION TITLE
Investigate Compiled Bindings

### DISCUSSION THREAD

**Discussion by cesarchefinho** [unoplatform/uno#12088] [Ideas]
Have anyone used https://github.com/levitali/CompiledBindings with UNO ?

Does this library contain any feature not supported by UNO x:bind implementation ?

Does this library contains any optimized code to improve UNO x:Bind implementation?

**Comment by jeromelaban** [comment]
Thanks for the suggestion! Uno's own implementation of bindings uses the syntax provided by WinUI, which is somehow different from what this nuget package provide (expressions are more complex).

If there's a WinUI implementation of this library, which means it may be feasible to create an uno version of it depending on the technique used to generate bindings.

**Reply by cesarchefinho** [reply]
> If there's a WinUI implementation of this library, 

Yes, it is an Winui Package but I don't know if it works with uno too.

**Reply by jeromelaban** [reply]
It's definitely not out of the box, no. But since it supports WinUI, it may be possible to compile it for Uno.
